### ISSUE TITLE
Open project up more to outside contributors

### ISSUE THREAD

**Issue by broofa** [npmgraph/npmgraph#56]
@fregante  I feel bad about how much I've dragged my feet on #50 and the other PRs you've put up.  It has me realizing that I should probably step back and let you run with this.  To that end, I'd like to propose the following...

* [x] Create an "NPMGraph" github org and add you as an "owner"
* [x] Transfer this repo to the new org
* [x] `npm owner add fregante` so you can publish to NPM as needed
* [x] Rename `master` branch to `main` (aligning with GH's new policy.  See [instructions](https://docs.github.com/en/github/administering-a-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes) for updating local repo branch name)
* [x] Setup github.io page(s) (See https://npmgraph.github.io)
* [x] Set up npmgraph.js.org (See https://github.com/js-org/js.org/pull/5788)
* [x] Redirect npm.broofa.com to ---^ (currently redirects to npmgraph.github.io.  Will redirect to npmgraph.js.org once that's up and running)

Thoughts?

**Comment by broofa** [comment]
> Create an "NPMGraph" github org

Alternatively, I could just add you as a collaborator to this project, but then it ends up published to broofa.github.io/npmgraph... 🤷

**Comment by styfle** [comment]
> I could just add you as a collaborator to this project, but then it ends up published to broofa.github.io/npmgraph

You can link this repo to vercel.com and then it will automatically deploy. Then you can configure a [custom domain](https://vercel.com/docs/custom-domains) to be something generic like https://npmgraph.vercel.app (although looks like someone already set that up 😄 )

**Comment by fregante** [comment]
Hi! The organization sounds good. The URL can either be the default .github.io or we can attempt to get npmgraph.js.org. 

As for npm I’m not sure why it’s published there since it’s an HTML app that can’t be `require`d. 

That vercel URL is actually what’s in #50 that I published to demo it 😃

**Comment by broofa** [comment]
> As for npm I’m not sure why it’s published there since it’s an HTML app that can’t be required.

Two reasons: 
1. So nobody else claims that name in NPM (I actually had someone squat on it as soon as I announced this project - 'ended up going through npm support to get ownership.)
2. Issues like #5, #9, and #19 point towards there being value in having a local server  and/or a CLI of some sort.

> we can attempt to get npmgraph.js.org

Great idea!

I've updated the issue description here to track the tasks for this.  Let me know if I'm missing anything.
